src/tagstudio/qt/ts_qt.py

```diff
@@ -1749,6 +1749,9 @@ def update_recent_lib_menu(self):
         """Updates the recent library menu from the latest values from the settings file."""
         actions: list[QAction] = []
         lib_items: dict[str, tuple[str, str]] = {}
+        filepath_option: str = str(
+            self.settings.value(SettingItems.SHOW_FILEPATH, defaultValue="show full path", type=str)
+        )
```

Comment:
*(In the context of enums)* Using a default settings value from an enum will also require needing `.value`, which isn't readily apparent so I wanted to make sure I mentioned it
```suggestion
        filepath_option: int = int(
            self.settings.value(
                SettingItems.SHOW_FILEPATH, defaultValue=ShowFilepathOption.DEFAULT.value, type=int
            )
```